spyder/plugins/editor/widgets/tests/test_folding.py

```diff
@@ -9,49 +9,69 @@
+    200: ('OK', 'Request fulfilled, document follows'),
+    201: ('Created', 'Document created, URL follows'),
+    202: ('Accepted','Request accepted, processing continues off-line'),
+    203: ('Non-Authoritative Information', 'Request fulfilled from cache'),
+    204: ('No Content', 'Request fulfilled, nothing follows'),
+    205: ('Reset Content', 'Clear input form for further input.'),
+    206: ('Partial Content', 'Partial content follows.'),
+    300: ('Multiple Choices','Object has several resources'),
+    301: ('Moved Permanently', 'Object moved permanently -- see URI list'),
+    302: ('Found', 'Object moved temporarily -- see URI list'),
+    303: ('See Other', 'Object moved -- see Method and URL list'),
+    304: ('Not Modified',
+        'Document has not changed since given time'),
+    305: ('Use Proxy',
+        'You must use proxy specified in Location to access this ',
+        'resource.'),
+    307: ('Temporary Redirect',
+        'Object moved temporarily -- see URI list'),
+
+    400: ('Bad Request',
+        'Bad request syntax or unsupported method'),
+    401: ('Unauthorized',
+        'No permission -- see authorization schemes'),
+    402: ('Payment Required',
+        'No payment -- see charging schemes')
+}"""
+
 
-# --- Fixtures
-# -----------------------------------------------------------------------------
 @pytest.fixture()
 def get_fold_levels():
     """setup editor and return fold levels."""
-    app = qapplication()
     editor = CodeEditor(parent=None)
     editor.setup_editor(language='Python')
 
-    text = ('# dummy test file\n'
-            'class a():\n'  # fold-block level-0
-            '    self.b = 1\n'
-            '    print(self.b)\n'
-            '    \n'
-            '    def some_method(self):\n'  # fold-block level-1
-            '        self.b = 3\n'
-            '\n'
-            '    def other_method(self):\n'  # fold-block level-1
-            '\n' # a blank line (should be ignored)
-            '  # a comment with arbitrary indentation\n' # should be ignored
-            '         a = (1,\n'  # fold-block level-2
-            '              2,\n'
-            '              3)\n'
-            )
+    lines_lvls = [0, 0, 1, 2, 1, 1, 1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+                  1, 1, 2, 1, 2, 2, 1, 2, 2, 1, 2, 1, 2, 1, 2, 0]
 
     editor.set_text(text)
-    return print_tree(editor, return_list=True)
+    return editor, lines_lvls
 
-# --- Tests
-# -----------------------------------------------------------------------------
-def test_simple_folding(get_fold_levels):
-    assert get_fold_levels == [[2, 0, 'V'],
-                                 [6, 1, 'V'],
-                                 [9, 1, 'V'],
-                                 [12, 2, 'V']]
 
+# --- Tests--------------------------------------------------------------------
+def test_simple_folding(get_fold_levels):
+    """Test folding by the levels."""
+    editor, lines_lvls = get_fold_levels
```

Comment:
Below this line please define a variable called `expected_folding_levels` and make it equal to the contents of what you called `lines_lvls` in the fixture.